nef-plugin/AppScheme.swift

```diff
@@ -1,41 +1,30 @@
 //  Copyright © 2019 The nef Authors.
 
 import AppKit
+import SourceEditorModels
 
 struct AppScheme {
-    enum Action {
-        case preferences
-        case carbon(selection: String)
-        case clipboardCarbon(selection: String)
-        case markdownPage(playground: String)
-        case playgroundBook(package: String)
-        
-        var item: URLQueryItem {
-            switch self {
-            case .preferences: return URLQueryItem(name: "preferences", value: nil)
-            case let .carbon(selection): return URLQueryItem(name: "carbon", value: selection)
-            case let .clipboardCarbon(selection): return URLQueryItem(name: "clipboardCarbon", value: selection)
-            case let .markdownPage(playground): return URLQueryItem(name: "markdownPage", value: playground)
-            case let .playgroundBook(package): return URLQueryItem(name: "playgroundBook", value: package)
-            }
-        }
-    }
+    let command: Command
+    let estimatedDuration: DispatchTime
```

Comment:
I am not convinced at all with the estimated duration, as we are not really able to estimate anything, so I'd suggest to completely remove it, or at least replace it by something more "honest".

Reply:
Ok, for now, we can remove it, and let see in the future if we need something more specific by each command.